lambdas/drivingpermitcheck/src/main/java/uk/gov/di/ipv/cri/drivingpermit/api/persistence/item/ContraindicationMappingItem.java

```diff
@@ -1,38 +1,9 @@
 package uk.gov.di.ipv.cri.drivingpermit.api.persistence.item;
 
 import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
-import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
-import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbSortKey;
 
 @DynamoDbBean
 public class ContraindicationMappingItem {
     private String thirdPartyId;
-    private String thirdPartyFraudCode;
     private String contraindicationCode;
-
-    @DynamoDbSortKey
-    public String getThirdPartyId() {
-        return thirdPartyId;
-    }
-
-    public void setThirdPartyId(String thirdPartyId) {
-        this.thirdPartyId = thirdPartyId;
-    }
-
-    @DynamoDbPartitionKey
-    public String getThirdPartyFraudCode() {
-        return thirdPartyFraudCode;
-    }
-
-    public void setThirdPartyFraudCode(String thirdPartyFraudCode) {
-        this.thirdPartyFraudCode = thirdPartyFraudCode;
-    }
-
-    public String getContraindicationCode() {
```

Comment:
Will need these for the CI mapping but I suppose can just add them back in on that ticket

Reply:
Yeah we can bring them back when needed